gradle/mockito-core/javadoc.gradle

```diff
@@ -52,6 +52,23 @@ javadoc {
     options.addStringOption('Xwerror', '-quiet')
     options.addBooleanOption('-allow-script-in-comments', true)
 
+    // attempt to fix https://github.com/mockito/mockito/issues/2765
+    doFirst {
```

Comment:
Let's put this in a separate task and add it as a dependency to the `javadoc` task here. Then, let's also add a verify task per https://github.com/michel-kraemer/gradle-download-task#verify-task to make sure we don't accidentally download malicious content from the internet.

Reply:
We can manually download the file, generate a shasum and then add that number as the verification sha. Then we know for sure it has the same file content as we expect and have read.

Reply:
> We can manually download the file, generate a shasum and then add that number as the verification sha.
true! 🤦‍♂️ 

thinking here... do we expect this `package-list` file to change? only if we move away from 8, right? why not just download it now and add it to the project?

Reply:
We don't expect it to change I think. In that case, yeah we can download it as well. Good point 😂

Reply:
BTW, what's the suggested place for these files? I was considering the below:

```
- gradle/mockito-core/java-8-docs/element-list
- gradle/mockito-core/java-8-docs/package-list
```

Reply:
Yeah that sounds good to me.